.circleci/workflows.yml

```diff
@@ -2817,6 +2845,13 @@ windows-workflow: &windows-workflow
         requires:
           - windows-build
 
+    - server-unit-tests-cloud-environment:
+        name: windows-server-unit-tests-cloud-environment
+        executor: windows
+        resource_class: windows.large
```

Comment:
not sure if this needs to be this big bc these are just unit tests

Reply:
Updated: [`e358a66` (#25837)](https://github.com/cypress-io/cypress/pull/25837/commits/e358a663e29e67a10a15cbd17f191d2f472d57b0)